src/include/cluster/dialogs.rb

```diff
@@ -1078,6 +1073,19 @@ def SecurityDialog
           _("Enable Security Auth"),
           true,
           VBox(
+            HBox(
+              HSpacing(20),
+              Left(ComboBox(
+                Id(:crypto_hash), Opt(:hstretch, :notify), _("Crypto Hash:"),
+                ["sha1", "sha256", "sha384", "sha512", "md5"]
+              )),
+              HSpacing(5),
+              Left(ComboBox(
+                Id(:crypto_cipher), Opt(:hstretch, :notify), _("Crypto Cipher:"),
+                ["aes256", "aes192", "aes128", "3des"]
```

Comment:
just note that values won't be translated. I am not sure if e.g. chinese has special characters for it or not.

Reply:
Do you mean "aes256", "3des" or "md5"? No, we also using the English word to represent it in Chinese world.